dags/public-health/Code55.py

```diff
@@ -0,0 +1,153 @@
+        "ian.rose@lacity.org",
+        "hunter.owens@lacity.org",
+        "brendan.bailey@lacity.org",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(days=1),
+}
+
+dag = DAG("code-55", default_args=default_args, schedule_interval="@daily")
+
+
+# Code to retrieve/clean data from each of the API endpoints
+def get_code_55_df(urlnumber, stage):
+    # make request to feature service url
+    url = (
+        r"https://production.sanstarla.com/FeatureService/REST/services"
+        "/Authorizations/FeatureServer/%s/query" % urlnumber
+    )
+    resp = requests.get(url)
+
+    # store raw data into JSON format
+    # isolate desired values from 'features' dict.keys()
+    raw_data = resp.json()
+    data = raw_data.get("features")
+    # create new empty list to store desired final data
+    cleandict = []
+    # iterate through 'features' dict.items() which are nested dictionaries
+    # isolate 'attributes' dict.items() and append to empty list
+    for d in data:
+        cleandict.append(d.get("attributes"))
+
+    # load list of dictionaries into a pandas dataframe and convert to CSV file
+    df = pd.DataFrame(cleandict)
+
+    # Deduplicating DataFrame
+    df["CompositeID"] = (
+        df["Number"].astype(str) + "-" + df["AssociatedServiceRequestNumber"]
+    )
+    df.drop_duplicates(subset="CompositeID", inplace=True)
+
+    # Assigning dataset
+    df["Stage"] = stage
+
+    # Adjusting datetimes from unix timestamp to readable date
+    datelist = [
+        "DateCreated",
+        "DateApproved",
+        "ExpirationDate",
+        "DateCompleted",
+        "CleanupDate",
+    ]
+    for date in datelist:
+        df[date] = pd.to_datetime(df[date], unit="ms")
+    return df
+
+
+def update_code_55(**kwargs):
+    # Collecting each dataset
+    code_55_closed_last_90 = get_code_55_df(0, "Closed")
+    code_55_pending_schedule = get_code_55_df(1, "Pending Scheduled")
+    code_55_scheduled = get_code_55_df(2, "Scheduled")
+
+    # Prioritizing data where Closed Last 90 > Scheduled > Pending Schedule
+    code_55_pending_schedule = code_55_pending_schedule.loc[
+        ~(
+            code_55_pending_schedule["CompositeID"].isin(
+                code_55_closed_last_90["CompositeID"]
+            )
+        )
+        & ~(
+            code_55_pending_schedule["CompositeID"].isin(
+                code_55_scheduled["CompositeID"]
+            )
+        )
+    ]
+    code_55_scheduled = code_55_scheduled.loc[
+        ~code_55_scheduled["CompositeID"].isin(code_55_closed_last_90["CompositeID"])
+    ]
+
+    # Merging all data into one dataframe
+    df = pd.concat(
+        [code_55_closed_last_90, code_55_pending_schedule, code_55_scheduled]
+    )
+
+    # Creating table if does not exist
+    engine = PostgresHook.get_hook("postgres_default").get_sqlalchemy_engine()
+    create_table_statement = """CREATE TABLE IF NOT EXISTS "public-health"."code55s" (
```

Comment:
Here and below can we also issue a `CREATE SCHEMA IF NOT EXISTS "public-health"` command?